AWS/anomaly_monitor/providers.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  experiments = [module_variable_optional_attrs]
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+provider "aws" {
```

Comment:
Provider block probably shouldn't be included (see [comment in other PR](https://github.com/mesoform/Multi-Cloud-Platform-Foundations/pull/14#discussion_r977500799))